.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - oraclejdk8
+  - openjdk8
```

Comment:
Just asking for an extra opinion here, @katmatt we had this problem https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/4 and had to switch to openjdk to fix it, is that fine with you ?

Reply:
That's fine with me. We anyway have to discuss how we want to proceed with supporting multiple JDK versions and distributions 😄

Reply:
cool, we ll this would solve our problems until we see we re the discussion in travis resolves.